docs/pom.xml

```diff
@@ -52,6 +44,83 @@
 			<id>docs</id>
 			<build>
 				<plugins>
+					<plugin>
+						<groupId>org.apache.maven.plugins</groupId>
+						<artifactId>maven-install-plugin</artifactId>
+						<version>3.1.1</version>
+						<executions>
+							<execution>
+								<id>spring-doc-resources</id>
+								<phase>clean</phase>
+								<goals>
+									<goal>install-file</goal>
+								</goals>
+								<configuration>
+									<file>src/lib/spring-doc-resources-0.2.2.RELEASE.zip</file>
```

Comment:
Why did you choose to put in `src/lib` rather than the `local-mavern-repo`? Aren't these supposed to be coming from the maven repository?

Reply:
These are not available in maven central. We used to get these from  `https://repo.spring.io/release`. According to recent changes repo.spring.io anonymous access is removed as of [5/25](https://status.spring.io/incidents/rl38k61jk6hr?u=dlh63rhg824z).
So as a temp workaround, I've downloaded these zip files from github to `src/lib`, and installing them to local repository with `maven-install-plugin` at `clean` phase for `docs` profile.